gammapy/spectrum/flux_point.py

```diff
@@ -821,10 +821,18 @@ def __init__(
     def ref_model(self):
         return self.model.model
 
+    @property
+    def e_groups(self):
+        """"""
+        from ..maps import MapAxis
+        e_edges = self.datasets.datasets[0].counts_on.energy.bins
```

Comment:
This really calls for `CountsSpectrum` to use a `MapAxis` rather than a `BinnedNDAxis`. But this will be for a later PR I can imagine.

Reply:
Yes, ideally we can reuse the `MapAxis` here in future.